docs/guides/01-css-elements.md

```diff
@@ -0,0 +1,127 @@
+# CSS Elements
+
+## Installation
+
+### From NPM
+  #### Install @carto/airship-style package from npm
+  ```
+  npm i @carto/airship-style --save
+  ```
+
+  #### Importing styles in SASS, PostCSS
+  ```css
+  // Import whole bundle
+  @import '~@carto/airship-style';
+
+  // Import SASS element styles
+  @import '~@carto/airship-style/src/{path}';
+
+  // Import CSS element styles
+  @import '~@carto/airship-style/dist/{path}';
+  ```
+
+  #### Importing styles in JavaScript
```

Comment:
should we mention that a specific loader is required?

Reply:
I don't know, I'd say that we can assume the reader knows how to do that if they use Webpack.

Reply:
I added a warning just in case any users feel lost about it.